README.md

```diff
@@ -1,8 +1,274 @@
-# RFC
+# Amethyst RFCs
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+[Amethyst RFCs]:
 
-## Creating a new RFC
+Note that this will be actual links once the bot is set up
+- discussing RFCs (should link to all open PRs here)
+- accepted RFCs (can link to the file tree)
+- RFCs waiting for implementation (issues with label="RFC accepted" -label="status: working"
 
-The specifics of the RFC process are not yet discussed, but will be documented in the first RFC.
+
+## Opening
+[Opening]: #opening
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Amethyst community and
+the [sub-team]s.
+
+The "RFC" (Request for Comments) process is intended to provide a consistent
+and controlled path for new features to enter the engine and associated
+libraries, so that all stakeholders can be confident about the direction in which
+the engine is evolving.
+
+You may join our Discord here: https://discord.gg/dqe9UYE
```

Comment:
If we are launching the forums at the same time as this, I would like to see it linked here too.

Reply:
Done